app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/service/state/RealVpnStateMonitor.kt

```diff
@@ -111,8 +110,9 @@ class RealVpnStateMonitor @Inject constructor(
         val runningState = when (lastState?.state) {
             ENABLING -> VpnRunningState.ENABLING
             ENABLED -> VpnRunningState.ENABLED
-            DISABLED -> VpnRunningState.DISABLED
-            else -> VpnRunningState.INVALID
+            DISABLED -> VpnRunningState.DISABLED()
+            SNOOZED -> VpnRunningState.DISABLED(TimeUnit.MINUTES.toMillis(20)) // TODO - remove hardcoded time
```

Comment:
Ideally we don’t have the hardcoded snooze length here. I wanted to remove it or replace it with a boolean, but this is currently where we get the snooze length when displaying the snooze length (https://github.com/duckduckgo/Android/pull/3869/files#diff-bb4ae2356a7050452a64491c94d8891321985471603d38084bdd0361ad741e86R129)

Still thinking how to improve this.